pom.xml

```diff
@@ -8,7 +8,7 @@
 
 	<artifactId>jigomerge</artifactId>
 	<packaging>hpi</packaging>
-	<version>0.9-SNAPSHOT</version>
+	<version>1.0</version>
```

Comment:
This line need not be changed. Once everything works, it will get updated automatically using maven release. Please change it back to the way it was.